docker/bactopia/requirements.txt

```diff
@@ -0,0 +1 @@
+PyYAML==6.0
```

Comment:
Can we install this dep via conda using `- pip`?

Reply:
Won't do